yarn-project/end-to-end/src/integration_l1_publisher.test.ts

```diff
@@ -144,6 +163,20 @@ describe('L1Publisher integration', () => {
     return tx;
   };
 
+  const forceInsertionInInbox = async (l1ToL2Messages: Fr[]) => {
+    for (let i = 0; i < l1ToL2Messages.length; i++) {
+      const slot = keccak256(Buffer.concat([l1ToL2Messages[i].toBuffer(), fr(0).toBuffer()]));
+      const value = 1n | ((2n ** 32n - 1n) << 128n);
+      // we are using Fr for the value as an easy way to get a correctly sized buffer and string.
+      const params = `["${inboxAddress}", "${slot}", "0x${new Fr(value).toBuffer().toString('hex')}"]`;
```

Comment:
I'd probably refactor this such that there is a method
```
const overwriteSlot = (address: EthAddress, slot: number, value: Buffer) => {
    // throw anvil stuff in here
}
```
^ this can be hidden away somewhere.  

Then have `forceInsertionIndex` be nice n tidy in this file.